### DISCUSSION TITLE
Add netherwhelp to store

### DISCUSSION THREAD

**Discussion by arcosin** [chromiecraft/chromiecraft#7099] [Suggestions]
Hello. Netherwhelp's Collar is normally awarded for buying collectors edition.
In line with other normally Blizz-awarded items, I think this should be available in the catalog. 
https://www.wowhead.com/wotlk/item=25535/netherwhelps-collar

**Reasons why:**
- Fundraising.
- Completeness.
- Netherwing faction is fkn great 🐉.

Thank you for the consideration 🙇

**Comment by dangeraaron10** [comment]
I asked on the discord if this pet was available ingame and was vaguely told "Yes, do challenges" yet no classes reward a Netherwhelp for any variant of Hardcore, Ironman, or Bloodthirsty.  At least, I didn't see it when checking the .challenges reward command.  Would like to see the Netherwhelp obtainable in some way as well.  It would be really nice to get one as a reward for grinding Netherwing reputation
